shaky-sample/src/main/java/com/linkedin/android/shaky/app/ShakyApplication.java

```diff
@@ -16,21 +16,50 @@
 package com.linkedin.android.shaky.app;
 
 import android.app.Application;
+import android.util.Log;
+
 import androidx.annotation.NonNull;
 import com.linkedin.android.shaky.EmailShakeDelegate;
 import com.linkedin.android.shaky.Shaky;
+import com.linkedin.android.shaky.ShakyFlowCallback;
 
 /**
  * Hello world example.
  */
 public class ShakyApplication extends Application {
+    private static final String TAG = ShakyApplication.class.getSimpleName();
 
     private Shaky shaky;
 
     @Override
     public void onCreate() {
         super.onCreate();
-        shaky = Shaky.with(this, new EmailShakeDelegate("hello@world.com"));
+        shaky = Shaky.with(this, new EmailShakeDelegate("hello@world.com"), new ShakyFlowCallback() {
+            @Override
+            public void onShakyStarted(@ShakyFlowCallback.ShakyStartedReason int reason) {
+                Log.d(TAG, "onShakyStarted: " + reason);
+            }
+
+            @Override
+            public void onShakyFinished(@ShakyFlowCallback.ShakeIgnoredReason int reason) {
+                Log.d(TAG, "onShakyFinished: " + reason);
```

Comment:
let's make the annotation consistent with the callback name here. @ShakyFlowCallback.ShakeyFinishedReason